tests/e2e/functional-e2e/functional_test.go

```diff
@@ -273,7 +273,7 @@ var _ = Describe("Functional e2e:", Serial, func() {
 		numPipelineVertices := len(updatedPipelineSpec.Vertices)
 		UpdatePipelineRollout(pipelineRolloutName, updatedPipelineSpec, numaflowv1.PipelinePhaseRunning, func(retrievedPipelineSpec numaflowv1.PipelineSpec) bool {
 			return len(retrievedPipelineSpec.Vertices) == numPipelineVertices
-		}, true, true)
```

Comment:
So, it's because we set `overrideSourceVertexReplicas` to true here that we need to set `scaleInVertex` to true down below, right?